### ISSUE TITLE
Labels for Build Coverage

### ISSUE THREAD

**Issue by zhangskz** [google/oss-fuzz#8975]
Labels configured in project.yaml do not seem to get automatically added to OSS-Fuzz reports for build coverage issues e.g.  https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=53283 is missing https://github.com/google/oss-fuzz/blob/14ec4885b45de9cc7a5a667bef7fdf3b9331b4db/projects/protobuf-python/project.yaml#L36

Other labels normally present (e.g. fuzzer, disclosure, etc) also do not seem to be present for this type of report as well.

**Comment by oliverchang** [comment]
Right, the build failure reports are a special case that do not get these labels applied. Is this important to your team's workflows?

**Comment by zhangskz** [comment]
We use these labels to ensure the reports get migrated to our bug component for our internal triage process. The lack of these labels for build coverage issues means that the emails for these are much more likely to be missed.
